src/common.jl

```diff
@@ -178,7 +178,7 @@ const FSCALES = (identity = identity, ln = log, log2 = log2, log10 = log10)  # f
 const ISCALES = (identity = identity, ln = exp, log2 = exp2, log10 = exp10)  # inverse
 const BASES = (identity = nothing, ln = "ℯ", log2 = "2", log10 = "10")
 
-const CRAYONS_FAST = Ref(true)
+const CRAYONS_FAST = Ref(false)
```

Comment:
What's the reasoning behind this change?

Reply:
Woops, my bad (development), will remove.